dev-guide/development.md

```diff
@@ -10,6 +10,8 @@ Before you begin, check the [supported platforms](./requirements.md#supported-pl
 
 You must follow [rust-rocksdb](https://github.com/pingcap/rust-rocksdb/blob/master/librocksdb_sys/build.sh#L127) to see which version TiKV needs. You can install the RocksDB shared library manually according to [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md).
 
+Once installed, please add the RocksDB include path to your `CPATH` environment variable and the RocksDB lib path to `LIBRARY_PATH`, `LD_LIBRARY_PATH` environment variables.
```

Comment:
Thanks @spongedu 

This is not a necessary way sometimes. If we install RocksDB into the `/usr/local/lib` or event `/usr/lib`, we don't need these.

Reply:
Yep. There's no need if they're already there.  I'll make some changes. PTAL